helm/charts/oathkeeper/values.yaml

```
@@ -193,6 +193,12 @@ secret:
   hashSumEnabled: true
 
 deployment:
+  strategy:
+    type: RollingUpdate
+    rollingUpdate:
```

I think a better approach here would be to have an empty object as a default value, to not enforce/provide concrete values. 
```suggestion
    rollingUpdate: {}
```

---

On the other hand, please also update the values in the `hack/values` directory with example values, and so that the CI will use and render them :)

---

Sorry for the delay. Right, done! Also see the other PR.